monin_obukhov/include/monin_obukhov.inc

```diff
@@ -123,6 +123,11 @@ do j=1, n3
   enddo
 enddo
 
+do j=1, n3
+  do i=1, n2
+    call stable_mix(rich(:, i, j), mix(:, i, j))
+  enddo
+enddo
```

Comment:
Hmm, this should probably be updated to the version of the file currently in the `mixedmode` branch.

Reply:
fixed!